packages/eslint-config/package.json

```diff
@@ -9,9 +9,14 @@
     "eslint": "^8.40.0",
     "eslint-config-next": "13.4.12",
     "eslint-config-prettier": "^8.8.0",
+    "eslint-config-turbo": "^1.9.3",
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-react": "^7.32.2",
-    "eslint-config-turbo": "^1.9.3",
     "typescript": "^5.1.6"
+  },
+  "devDependencies": {
+    "eslint-plugin-import": "^2.28.1",
+    "eslint-plugin-package-json": "^0.1.4",
+    "eslint-plugin-simple-import-sort": "^10.0.0"
```

Comment:
We do this via prettier plugins now :D

Reply:
The `import-sort` or the `package-json` one? 👀

Reply:
@Mythie Could you please let me know about this? So that I can wrap this PR up